### PR TITLE
Removed unnecessary "break"

### PR DESCRIPTION
**Pull request by ExactlySeroja**
Just a minor code improvment

### CODE REVIEW

File: src/main/java/org/example/Service.java [hunk 1]
```diff
@@ -65,7 +65,6 @@ protected void showTable() throws SQLException {
             tableNumber = scanner.nextInt();
             if ((tableNumber >= 1) && (tableNumber <= 4)) {
                 check = false;
-                break;
```
Comment (k4rnaj1k) [inline src/main/java/org/example/Service.java:68]: Is it _necessary_?

File: src/main/java/org/example/Service.java [hunk 2]
```diff
@@ -65,7 +65,6 @@ protected void showTable() throws SQLException {
...
             } else
                 System.out.println("Wrong action! Try again");
```
Comment (k4rnaj1k) [inline src/main/java/org/example/Service.java:69]: looks weird

File: src/main/java/org/example/Service.java (hunk 1 above)
Comment (ExactlySeroja) [reply src/main/java/org/example/Service.java:68]: 100% necessary

File: src/main/java/org/example/Service.java (hunk 2 above)
Comment (ExactlySeroja) [reply src/main/java/org/example/Service.java:69]: It's required for our program

**Review by k4rnaj1k — APPROVED** [review]
Good job

**Pull request merged**